### DISCUSSION TITLE
CUSA28561 Horizon Forbidden West (US)

### DISCUSSION THREAD

**Discussion by iioriyagami** [GoldHEN/GoldHEN_Cheat_Repository#56] [Cheat requests]
Infinite health (master code)
1hit kill (master code) 

Doesn't work. PS4 9.00.

**Comment by kmeps4** [comment]
Hello there:

On GoldHEN side we do not create/develop cheats... if you need to ask an exclusive cheat, please redirect to: https://twitter.com/PS4Trainer and join their Discord Server and ask for needed Cheat.

If you wanna Submit a PS2/PS4 cheat you have created, please go to GoldHEN discord: https://discord.gg/pwfKbWzEx3 and submit them on the respective Cheat channel.

Have a great one!!!
